desktop/core/src/desktop/js/reactComponents/TaskBrowser/TaskBrowser.tsx

```diff
@@ -0,0 +1,434 @@
+          <option key={taskName} value={taskName}>{taskName}</option>
+        ))}
+      </select>
+      </div>
+      <div className='vertical-spacer'></div>    
+      {selectedTask && (
+        <div className="parameter-inputs">
+        {tasks[selectedTask].params.map((param) => (
+          <div key={param.name} className="parameter-row">  
+          <label className="parameter-label">{param.name}:</label>
+          <input
+          name={param.name}
+          type="text"
+          placeholder={param.name}
+          onChange={handleChange} // Use handleChange for input changes
+          value={params[param.name] || ''}
+          className="parameter-input"
+        />
+        </div>
+      ))}
+      </div>
+    )}
+    </Modal>    
+  );
+};
+
+const TaskBrowser = () => {
+  const [showSchedulePopup, setShowSchedulePopup] = useState(false);
+  const [scheduledTasks, setScheduledTasks] = useState([]); // Store an array of scheduled tasks
+
+  const [showLogPopup, setShowLogPopup] = useState(false); // This should match where you use it
+  const [currentTaskId, setCurrentTaskId] = useState(null);
+  const [taskLogs, setTaskLogs] = useState("");
+
+  const handleScheduleSubmit = (taskName, taskParams) => {
+    const payload = {
+      taskName,
+      taskParams,
+    };
+
+    axios.post('/desktop/api2/handle_submit', payload)
+    .catch(error => {
+      console.error('Error scheduling task', error);
+    });
+  };
+
+  // This function will open the schedule task popup
+  const handleSchedulePopup = () => {
+    setShowSchedulePopup(true);
+  };
+
+  // Add the function to handle showing logs
+  const handleShowLogs = (taskId) => {
+    // This function should set the current task ID and fetch the logs
+    console.log("hi im in handlelogs");
+    setCurrentTaskId(taskId);
+    // Fetch the logs from the server
+    axios.get(`/desktop/api2/get_task_logs/${taskId}/`)
+      .then(response => {
+        // Assuming the server response contains the logs in the data object
+        setTaskLogs(response.data);
+        setShowLogPopup(true); // Open the popup
+      })
+      .catch(error => {
+        console.error('Error fetching logs:', error);
+        // Handle the error appropriately
+      });
+  };
+
+  return (
+    <div>
+      {showSchedulePopup && (
+        <ScheduleTaskPopup
+          onClose={() => setShowSchedulePopup(false)}
+          onSubmit={handleScheduleSubmit}
+          open={showSchedulePopup}
+        />
+      )}
+      <div style={{ margin: '20px 0' }}>
+        <TaskBrowserTable tasks={scheduledTasks} handleShowLogs={handleShowLogs} handleSchedulePopup={handleSchedulePopup}/>
+      </div>
+      {showLogPopup && (
+        <div className="popupStyle">
+          <div className="popupContentStyle">
+            <h2>Task Logs - {currentTaskId}</h2>
+            <pre>{taskLogs}</pre>
+            <button onClick={() => setShowLogPopup(false)}>Close</button>
```

Comment:
[Ant button](https://ant.design/components/button)

Reply:
Nit: The cuix buttons are actually 100% aligned with the Cloudera Design Language spec so we should use them when possible over the ant buttons.